peering/peering.go

```diff
@@ -0,0 +1,325 @@
+package peering
+
+import (
+	"context"
+	"errors"
+	"math/rand"
+	"strconv"
+	"sync"
+	"time"
+
+	"github.com/ipfs/go-log"
+	"github.com/libp2p/go-libp2p/core/host"
+	"github.com/libp2p/go-libp2p/core/network"
+	"github.com/libp2p/go-libp2p/core/peer"
+	"github.com/multiformats/go-multiaddr"
+)
+
+// Seed the random number generator.
+//
+// We don't need good randomness, but we do need randomness.
```

Comment:
I guess it's not you, I think this was just copied over but what is that ? Did you modfied something or previous outdated comments as it looks like to be ?

Reply:
Previous outdated commits. This was nothing I did.